ksqldb-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -214,12 +213,15 @@ public Map<String, TopicDescription> describeTopics(final Collection<String> top
           ).allTopicNames().get(),
           ExecutorUtil.RetryBehaviour.ON_RETRYABLE);
     } catch (final ExecutionException e) {
+      LOG.error("Failed to Describe Kafka Topic(s): {}", topicNames, e);
       throw new KafkaResponseGetFailedException(
           "Failed to Describe Kafka Topic(s): " + topicNames, e.getCause());
     } catch (final TopicAuthorizationException e) {
+      LOG.error("Topic authorization failed for topics: {}", topicNames, e);
       throw new KsqlTopicAuthorizationException(
           AclOperation.DESCRIBE, topicNames);
     } catch (final Exception e) {
+      LOG.error("Failed to Describe Kafka Topic(s): {}", topicNames, e);
```

Comment:
This log line can be different than the previous one to help differentiate the exact exception.